contribs/src/main/java/com/netflix/conductor/contribs/publisher/TaskNotification.java

```diff
@@ -45,6 +46,10 @@ public String getAccountMoId() {
         return accountMoId;
     }
 
+    public String getWorkflowInfoMoId() {
```

Comment:
Why we need WorkflowInfoMoId as part of  Task/Wf notification ?

Reply:
done